packages/js/components/src/pill/pill.js

```diff
@@ -9,7 +9,7 @@ import classnames from 'classnames';
  */
 import { Text } from '../experimental';
 
-export function Pill( { children, className } ) {
+export function Pill( { children, className = '' } ) {
```

Comment:
I think we don't need to add a changelog for this because `className` is just introduced and not released yet.